internal/testing/backend/search_test.go

```diff
@@ -0,0 +1,143 @@
+//go:build integration
+
+package backend_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/guacsec/guac/internal/testing/testdata"
+	"github.com/guacsec/guac/pkg/assembler/graphql/model"
+)
+
+func TestFindSoftware(t *testing.T) {
+	b := setupTest(t)
+	tests := []struct {
+		name       string
+		searchText string
+		want       []model.PackageSourceOrArtifact
+		wantErr    bool
+		InPkg      []*model.PkgInputSpec
+		InSrc      []*model.SourceInputSpec
+		InArt      []*model.ArtifactInputSpec
+	}{
+		{
+			name:       "default package",
+			InPkg:      []*model.PkgInputSpec{testdata.P1},
+			searchText: "tensorflow",
+			want: []model.PackageSourceOrArtifact{
+				model.Package{
```

Comment:
i think we have a helper in internal called `pkg` that will help make this more readable.

Reply:
so there is an output already constructed here for `testdata.P1`: https://github.com/guacsec/guac/blob/main/internal/testing/testdata/models.go#L155-L166

Reply:
I replaced the wants with the helper, thank you @lumjjb and @pxp928